client/src/components/OnboardUser.jsx

```diff
@@ -96,6 +97,7 @@ const OnboardUser = props => {
   const { setDemographicsComorbiditiesThunk } = props;
   const { userSession } = useBlockstack();
   const [formState, setFormState] = useState(blankForm);
+  const [userWantsToDelete, setuserWantsToDelete] = useState(false);
```

Comment:
This is nitpicky, only change if you feel the need, but maybe change the name to something a little less "action-y" - not sure else how to describe it. Something like `[deleteAllFiles, setDeleteAllFiles]`?

Reply:
I'll throw in another idea:

`[showDeletionDialog, setShowDeletionDialog]`